opendbc/car/ford/radar_interface.py

```diff
@@ -55,7 +56,9 @@ def update(self, can_strings):
     vls = self.rcp.update_strings(can_strings)
     self.updated_messages.update(vls)
 
-    if self.trigger_msg not in self.updated_messages:
+    # print('updated_messages:', self.updated_messages)
+    # if self.trigger_msg not in self.updated_messages:
+    if 368 not in self.updated_messages:
```

Comment:
The LSB scan index signals always match the header counter's 2 LSB when waiting for it now, before it would usually have a few messages that mismatched